index.Rmd

```diff
@@ -35,15 +37,26 @@ card(
 
 ## Installation
 
+You can install {shinyMobile} in the usual ways:
+
 ```r
 # from CRAN
 install.packages("shinyMobile")
+
 # for the latest version
 devtools::install_github("RinteRface/shinyMobile")
 ```
 
 ## Demo
```

Comment:
@DivadNojnarg for the Android and iOS demo we'll wait for the new Gallery app

Reply:
The router layout does not work on shinylive. So the showcase will still use the existing code with some improvements.